Firestore/Example/Tests/GoogleTest/GoogleTest.podspec

```diff
@@ -0,0 +1,84 @@
+# Copyright 2017 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A Private podspec for GoogleTest. Suitable only for use inside this source
+# tree.
+
+Pod::Spec.new do |s|
+  s.name             = 'GoogleTest'
+  s.version          = '1.8.0'
+  s.summary          = 'Google Test'
+
+  s.description      = <<-DESC
+Google's C++ test framework.
+                       DESC
+
+  s.homepage         = 'https://github.com/google/googletest/'
+  s.license          = 'BSD'
```

Comment:
Might want to get https://github.com/google/googletest/ to make their licensing clearer with a LICENSE file in their repo.

Reply:
There are two, one for [googletest](https://github.com/google/googletest/blob/master/googletest/LICENSE) and one for [googlemock](https://github.com/google/googletest/blob/master/googlemock/LICENSE).

Reply:
I can't find documentation about how to handle more than one license for a pod. My recommendation would be to make it `s.license = { :type => 'custom', :file => 'LICENSE' }` and put pointers to the two licenses in the LICENSE file.

Reply:
The issue here is that neither file actually exists here. This is a local podspec for a remote repository.